community-operators/spark/spark.package.yaml

```diff
@@ -0,0 +1,5 @@
+#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/etcdoperator.v0.9.2.clusterserviceversion.yaml
+packageName: spark
```

Comment:
Not sure if this should be called `spark` or something more specific. There's no conflict with the other spark operator, but I feel that naming this one `spark` suggests this is more official than others, or that this is the original spark operator.

Reply:
Yes it does make it seem like that.
How about `gcp-spark`, even though it's not an official product or has any other branding that I can see on the repo.
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator

Reply:
I think that's fine. It's from the GCP GitHub org so it's clear and accurate.